src/sentry/models/organizationonboardingtask.py

```diff
@@ -150,6 +152,7 @@ class OrganizationOnboardingTask(AbstractOnboardingTask):
             OnboardingTask.METRIC_ALERT,
             OnboardingTask.INTEGRATIONS,
             OnboardingTask.SESSION_REPLAY,
+            OnboardingTask.FIRST_PROFILE,
```

Comment:
Should profiling be on the list of required tasks? We don't support every platform.

Reply:
@Zylphrex We have a few JS only onboarding tasks so I guess that's fine? If we need to fix this we can do it holistically.